cloudbuild.yaml

```diff
@@ -1,18 +1,20 @@
 # See https://cloud.google.com/cloud-build/docs/build-config
 options:
+  dynamic_substitutions: true
   substitution_option: ALLOW_LOOSE
   machineType: 'N1_HIGHCPU_8'
 steps:
   - name: gcr.io/k8s-testimages/gcb-docker-gcloud
     entrypoint: /buildx-entrypoint
     args:
     - build
-    - --tag=gcr.io/$PROJECT_ID/cloud-controller-manager:$_GIT_TAG
+    - --tag=gcr.io/$PROJECT_ID/cloud-controller-manager:$_SHORT_TAG
     - --tag=gcr.io/$PROJECT_ID/cloud-controller-manager:latest
-    - --build-arg=VERSION=$(echo $_GIT_TAG | cut -d - -f2-)
+    - --build-arg=VERSION=$_SHORT_TAG
     - --output=type=registry
     - --platform=linux/amd64,linux/arm64
     - .
 substitutions:
-  _GIT_TAG: '12345'
+  _GIT_TAG: 'v99999999-v12345'
+  _SHORT_TAG: '${_GIT_TAG:10}'
```

Comment:
for reference: https://cloud.google.com/build/docs/configuring-builds/use-bash-and-bindings-in-substitutions#bash_parameter_expansions

Reply:
Is it possible to have a comment here saying something like `# Remove date prefix (first 10 characters) to create valid semver version, e.g. v20220510-v1.24.0-alpha.0-15-g09bd268 => v1.24.0-alpha.0-15-g09bd268`